httpstats/handler.go

```diff
@@ -118,5 +118,5 @@ func (w *responseWriter) complete() {
 	}
 
 	w.metrics.observeResponse(res, "write", w.bytes, now.Sub(w.start))
-	w.eng.ReportAt(w.start, w.metrics)
+	w.eng.ReportAt(w.start, w.metrics, Tags(w.req)...)
```

Comment:
This would be more efficient to not take the copy here and reference it directly.  Probably safe enough since this is all in the same package.